.vscode/launch.json

```diff
@@ -13,10 +13,6 @@
       "args": [],
       "cwd": "${workspaceFolder}/ntbs-service",
       "stopAtEntry": false,
-      "serverReadyAction": {
```

Comment:
My impression was that having debugger launch a new browser window on each restart was annoying - let me know if that's not the case!